plugins/ccgx/ccgx-dock.quirk

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright (C) 2020 Cypress Semiconductor Corporation.
+#
+# SPDX-License-Identifier: LGPL-2.1+
+#
+
+#######################################################
+## Gen2 Dock
+#######################################################
+[DeviceInstanceId=USB\VID_17EF&PID_A38F]
+Name = Lenovo Gen2 Dock Billboard
+Plugin = ccgx
+GType = FuCcgxDockBb
+ParentGuid = USB\VID_17EF&PID_A392
```

Comment:
shouldn't this be `cy-device-external-bb` too?

Reply:
USB\VID_17EF&PID_A38F is Billboard device but I set the cy-device-external-bb flag to [Guid=USB\VID_17EF&PID_A38F&bb].

Could you let me know what is the difference between [DeviceInstanceId=...] and [Guid=...]?